css/mirador.less/panels/bottom-panel.less

```diff
@@ -38,6 +38,12 @@
         padding: 0px 10px 0px 10px;
         /*transition: all 0.03s ease-out;*/
         display: inline-block;
+        margin-bottom: 11px !important;
+
+        &.highlight {
+          padding-left: 8px !important;
+          padding-right: 8px !important;
```

Comment:
Can we find a way to do this without `!important`.

Some opinions:
https://css-tricks.com/when-using-important-is-the-right-choice/
http://stackoverflow.com/a/3706876/564684

In this case, the specificity is not narrow enough for the change to take hold. If this is how we want to fix the bump up, we should find out why it's happening and reorganise accordingly. Our strategy for attaching the label seems to be what's causing the problem here. The behaviour we probably intended was: "make all thumbnails line up at the top as equal height, and display the label underneath. If the label is too long, truncate it. If there is no label, the thumbnails should still line up at the top, and there should simply be a blank space beneath those without labels.